docs/guide/usage/mqtt_topics_and_messages.md

```diff
@@ -6,6 +6,14 @@ sidebarDepth: 1
 
 This page describes which MQTT topics are used by Zigbee2MQTT. Note that the base topic (by default `zigbee2mqtt`) is configurable in the [Zigbee2MQTT `configuration.yaml`](../../guide/configuration/). In Zigbee2MQTT 1.17.0 a new API was introduced, the documentation of the legacy api can be found [here](https://github.com/Koenkk/zigbee2mqtt.io/blob/ead922ee141546ccce079430a7acce67c982c99b/docs/information/mqtt_topics_and_messages.md).
 
+## Publihing messages
```

Comment:
I think it makes more sense to move this to `zigbee2mqtt/FRIENDLY_NAME/set`, e.g. after the without JSON section:


### Publishing messages
Publishing messages depends on the MQTT client you use. For example to publish a message using the command line with Mosquitto you can use the command

```bash
 mosquitto_pub -t 'zigbee2mqtt/DEVICE_FRIENDLY_NAME/set' -m '{ "state": "ON"}'
```

Reply:
Done